<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anritsu Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxConnection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QComboBox" name="comboPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zeroButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Zero</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QLabel" name="labelFrequency">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency (MHz)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFrequency">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSetFrequency">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAttenuation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Attenuation (dB)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAttenuation">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStartStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSetAttenuation">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>291</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Power</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPowerValue">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>- dBm</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWattValue">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>- W</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAttenuation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Attenuation Table</string>
    </property>
    <widget class="QPushButton" name="buttonLoadCsv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Load CSV</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLoadPresets">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Presets</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCsvStatus">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>No CSV file loaded</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboPresets">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>180</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelStartFreq">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Freq (MHz)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditStartFreq">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelEndFreq">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>85</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>End Freq (MHz)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEndFreq">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonCalculateAttenuation">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate Avg</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>